docs/Developers/Newcomers_Guide.rst

```diff
@@ -1,6 +1,12 @@
 Welcome to the Newcomers' Guide!
 ================================
 
+.. container:: shorturl
+
+    +-----------+---------------------------+-----------------------------+
+    | Short-URL | https://coala.io/newcomer | https://coala.io/newcomers  |
+    +-----------+---------------------------+-----------------------------+
```

Comment:
Maybe call it "Redirects from:" / "Redirects from" (w/o colon) ? Like on a wiki page that allows many redirections.
Up to you :)

Reply:
Just in case: What if we would have more URLs so we get a line-break? Doe we want to do one cell -> one link like here? Or maybe one cell for all links, separated by whitespace? Up to you, only want to discuss the layout, since knowing the short URLs to those pages is not necessarily the information one seeks when opening that page ;)

Reply:
Actually, I got the idea of the position from this comment: https://github.com/coala/coala/issues/3754#issuecomment-366485906 . Please suggest me an appropriate placement. :)

Reply:
Not completely sure myself, that's why I just question it ;)